modules/33acrossBidAdapter.js

```diff
@@ -85,6 +83,26 @@ function _validateVideo(bid) {
     return false;
   }
 
+  const placement = videoAdUnit.placement || videoBidderParams.placement
```

Comment:
i think its better to access the `placement` and `startdelay` in one place. like just do it at bidderParams, also i didnt see any example that has those variables set under videoAdUnit. When the pub start setting these params everywhere, it will be hard for our Ops team to help out. plus we access this variable one by one, it will create a merge situation here eg. `Object.assign({}, videoBidderParams, videoAdUnit)`, its hard for us to debug any pubsup ticket

Reply:
So in [Prebid Docs](https://docs.prebid.org/dev-docs/bidder-adaptor.html#step-2-accept-video-parameters-and-pass-them-to-your-server) they are recommending that pubs set video params in Ad Unit itself. Only set in bidder params they wish to override for us. I am just leaving open the possibility for the Pub to set bidder specific overrides. There's a lot of variation between other adapters in how they enforce it. Of course, also in Prebid 4.0+ they are asking us to look first Ad Unit and then in bidder params. The code reflect this recommendation here and in `_buildVideoORTB`

Having said the above, I agree that for the sake of PUBSUP / Ops support we should advise the Pubs to set it in Ad Unit. The official docs will do what Prebid is asking them to do i.e. set video params under Ad Unit.